eox_nelp/pearson_vue/rti_backend.py

```diff
@@ -220,6 +220,7 @@ class ResultNotificationBackend(RealTimeImport):
     for handling result notifications, including extracting result notification data,
     retrieving enrollment information, and generating external certificates.
     """
+    use_audit_backend = False
```

Comment:
Do you thinks is not important to audit  ResultNotificationBackend
![image](https://github.com/user-attachments/assets/6e2cf87c-607f-46a7-83be-1581f5a220c5)

Reply:
I think that is important but it's not possible, anyway, the certificate has its own audited method so we won't be able to audit the whole pipeline but we will get the same current behavior for external certificates